docs/device_api.md

```diff
@@ -281,12 +267,9 @@ $ mosquitto_pub -h $HOST_IOTAGENT_MQTT -u theUser -P thePassword -t /<api_key>/m
 The simple JSON protocol used by the JSON IoTAgent maps each measurement to an attribute in a JSON Object. The following
 example shows how to send a measurement of three different quantities:
 ```
-POST  $HOST_IOTAGENT/iot/json?k=<apikey>&i=<device_ID>
-Headers:
-{
-	"content-type": "text/plain"
-}
-Payload:
+POST  /iot/json?k=<apikey>&i=<device_ID>
+Content-type: text/plain
```

Comment:
I have keep the original value of text/plain for this header but, looking below the hader, I wonder if this should actually be application/json.

Reply:
Fixed in htps://github.com/telefonicaid/fiware-iot-stack/pull/72/commits/1cf93c7c7ae54ed29c57d8d04bbe71268ff3a434